### ISSUE TITLE
No more IPC?

### ISSUE THREAD

**Issue by Cazzar** [AsgardXIV/Brio#15]
I was in the process of developing an [additional tool](https://github.com/Cazzar/GposeUtils) that was utilising Brio as a quick way of spawning minions.
With the release of Brio 0.2 I have noticed that the IPC has been completely removed and the only way to achieve this would be interacting with the HTTP API, which within the scope of a Dalmund plugin, that seems a bit excessive.

Is there a chance of getting the IPC API re-introduced, so I can continue the work on this?

Thanks.

**Comment by AsgardXIV** [comment]
Oh, my apologies. No one was using it so I just didn't port it when I did the rework.
Any there specific APIs you are interested in, was the existing surface enough?

**Comment by Cazzar** [comment]
No worries, the existing ones were plenty for a basic setup. Though I did have a few things that might be easier to also support at the IPC level, as I had to work around them myself, though none of these would be specifically "need-to-do" items

1. Allow spawning an actor without cloning gear/outfit
2. Allow spawning of an arbitrary model ID (this was achieved on my side with `((Character*)brioObject.Address)->CharacterData.ModelCharaId = modelId;`
3. Allow spawning at a specific x, y, z

No. 3 was always the most tricky for me to work out, and the current implementation doesn't really work fully when posing mode is enabled in the likes of Ktisis.

If you want a high level scope of what I am making, is a tool that would allow someone while posing to quickly spawn commonly used minions for example, sun/moon/leviathan.
